src/docs/quick-start.rst

```diff
@@ -1,12 +1,182 @@
+===========
 Quick Start
 ===========
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ac ligula et felis aliquam posuere. Etiam ac augue vel tortor porttitor placerat vitae ac libero. Nunc a efficitur lectus. Aliquam malesuada congue arcu quis luctus. Quisque pretium dolor orci, ut ullamcorper felis tincidunt eget. Duis auctor felis ut maximus facilisis. Nam eu feugiat tortor.
+This page will provide brief instructions to configure SSSD with
+FreeIPA, AD, and LDAP.
+
+Quick Start IPA
+***************
+
+Before starting, make sure you have the following information.
+
+- Administrator credentials *i.e. admin*
+- FreeIPA domain name *i.e. sssd.io*
+- FreeIPA server hostname *i.e. ipa1.sssd.io*
+- FreeIPA server IP *i.e. 1.2.3.4*
+
+Install the necessary packages, for RHEL and clones the package is named **ipa-client**, and for Fedora it’s **freeipa-client**.
```

Comment:
ReST uses double backticks for \``inline-code``. Please use inline code for commands, file names and values. This applies to the whole document.